ably/testutil/test_app.go

```diff
@@ -64,24 +73,32 @@ func (t *App) populate(res *http.Response) error {
 		return err
 	}
 
-	t.Options.Key = t.AppKeyId() + ":" + t.AppKeyValue()
+	t.opts.Token = t.AppKeyId()
```

Comment:
opts.Token should be opts.ApiKey and t.AppKeyID should be t.ApiSecret, is it right?